test/jdk/javax/swing/JButton/4659800/EnterKeyActivatesButton.java

```diff
@@ -87,30 +89,32 @@ private void createUI() {
     public void runTest() throws Exception {
         Robot robot = new Robot();
         robot.setAutoDelay(100);
-        for (UIManager.LookAndFeelInfo laf : UIManager.getInstalledLookAndFeels()) {
+        //Filter out Windows related LnFs.
```

Comment:
The comment is misleading: Filter out non-Windows L&Fs.

Usually, there's a space after `//`.